cmd/hackenv/main.go

```diff
@@ -30,6 +31,9 @@ func init() {
 }
 
 func main() {
+	if cmd.Verbose {
+		log.SetLevel(log.DebugLevel)
+	}
```

Comment:
This should come after the call to `kong.Parse()`, because before `cmd` is not properly populated.